pkg/specgen/generate/container.go

```diff
@@ -156,7 +156,9 @@ func CompleteSpec(ctx context.Context, r *libpod.Runtime, s *specgen.SpecGenerat
 
 		// Add annotations from the image
 		for k, v := range inspectData.Annotations {
-			annotations[k] = v
+			if !define.IsReservedAnnotation(k) {
```

Comment:
Seems like it would be at least worth warning about completely ignoring a user-defined annotation?

Reply:
A warning seems too aggressive IMO since it's not defined by the user but by the image. Note that Docker does not apply image annotations to the containers.

Cool with a debug message?

Reply:
Sure